.github/workflows/rust.yml

```diff
@@ -170,7 +170,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ test, test-no_std, test-wasm ]
+        target: [ test, test-no_std, test-wasm, test-lambdaworks-felt ]
```

Comment:
We need the backend to be a different argument. Otherwise we don't test the combination of different runtimes (or lack thereof).

Reply:
For the meantime, would adding `test-lambdaworks-felt-no-std` & `test-lambdaworks-felt-wasm `suffice?